pkg/embeddedcluster/node_join.go

```diff
@@ -187,9 +187,13 @@ func GenerateK0sJoinCommand(ctx context.Context, client kubernetes.Interface, ro
 
 // gets the port of the 'admin-console' service
 func getAdminConsolePort(ctx context.Context, client kubernetes.Interface) (int32, error) {
-	svc, err := client.CoreV1().Services(util.PodNamespace).Get(ctx, "admin-console", metav1.GetOptions{})
+	svc, err := client.CoreV1().Services(util.PodNamespace).Get(ctx, "kurl-proxy-kotsadm", metav1.GetOptions{})
```

Comment:
just noticed that it the error returned in the last line of the function still references the old name

Reply:
I'll fix it